core/src/main/java/com/klaytn/caver/tx/account/AccountKeyRoleBased.java

```diff
@@ -108,4 +108,13 @@ public boolean equals(Object o) {
         return Arrays.equals(toRlp(), that.toRlp());
     }
 
+    @Override
+    public String toString() {
+        StringBuilder result = new StringBuilder();
```

Comment:
I'm not sure this is the right time to tell you, but what about using JSON mashalling instead of making raw strings?

Reply:
Why should I marshaling JSON? I'll work through https://groundx.atlassian.net/browse/TPT-1055 if needed.

Reply:
It's because it is more common and comprehensive way of doing this. Actually, the format that you used here is very similar to JSON. Also, it is much more easy to adapt if we need a new field. By the way, please just move forward. This could be done later. :)